alarm/uboot-veyron/PKGBUILD

```diff
@@ -4,36 +4,48 @@
-pkgname=('uboot-minnie' 'uboot-mickey' 'uboot-jerry')
-pkgver=2017.11
+pkgname=('uboot-minnie' 'uboot-mickey' 'uboot-jerry' 'uboot-speedy')
+pkgver=2018.07
 pkgrel=1
 arch=('armv7h')
 url="http://git.denx.de/u-boot.git/"
 license=('GPL')
-makedepends=('git' 'bc' 'dtc' 'vboot-utils' 'python2' 'swig')
+makedepends=('bc' 'dtc' 'vboot-utils' 'python2' 'swig')
 conflicts=('linux-veyron' 'linux-armv7-chromebook')
 install=${pkgbase}.install
 source=("ftp://ftp.denx.de/pub/u-boot/u-boot-${pkgver/rc/-rc}.tar.bz2"
-        '0001-rk3288-set-CONFIG_SYS_TEXT_BASE-for-depthcharge.patch'
-        '0002-rockchip-minnie-Add-a-.its-file-for-chromium.patch'
-        '0003-rockchip-mickey-Add-a-.its-file-for-chromium.patch'
-        '0004-dtoc-python2-compatibility.patch'
+        '0001-rockchip-add-support-for-veyron-speedy-ASUS-Chromebo.patch'
+        '0002-rockchip-fix-incorrect-detection-of-ram-size.patch'
+        '0003-Revert-doc-fix-incorrect-usage-of-DT-node-unit-addre.patch'
+        '0004-rockchip-minnie-Add-a-.its-file-for-chromium.patch'
+        '0005-rockchip-mickey-Add-a-.its-file-for-chromium.patch'
+        '0006-rockchip-speedy-Add-a-.its-file-for-chromium.patch'
+        '0007-veyron-set-CONFIG_SYS_TEXT_BASE-for-depthcharge.patch'
+        '0008-dtoc-python2-compatibility.patch'
         'boot.txt'
         'mkscr')
-md5sums=('7848f6fd197e2c6ef2eb483dd5bd2858'
-         '1ff4ad552e1c5d32ff0a561f2d6ac702'
-         'a7560967318753fbb2a60b0e15c5254d'
-         '612a0be69d8d55187ffe1782802cf566'
-         'b2080e9a988a5be5356ef1f782ec23f2'
+md5sums=('2b8eaa30dd118b29889669070da22bb0'
+         '315442388197340887024e447638ae89'
+         '9eac97ec89da2198997b638ba314f23f'
+         '806b653be429fe4e5428afb51ea38798'
+         'cb53652b687a129390947639dfeb9a08'
+         '983b5796b9858f8f8a148b4438eb18a7'
+         'efeb8eb39a41a1fe8ab32ad602239a1b'
+         '3268916dba41194d44f4c1e61b739288'
+         '489abfae2a3a6cb33c8e31433d533de1'
          '029cb473f59f08834b54cdc3e34c9672'
          '021623a04afd29ac3f368977140cfbfd')
 
 prepare() {
   cd u-boot-${pkgver/rc/-rc}
-  git apply ../0001-rk3288-set-CONFIG_SYS_TEXT_BASE-for-depthcharge.patch
-  git apply ../0002-rockchip-minnie-Add-a-.its-file-for-chromium.patch
-  git apply ../0003-rockchip-mickey-Add-a-.its-file-for-chromium.patch
-  git apply ../0004-dtoc-python2-compatibility.patch
+  patch -Np1 -i ../0001-rockchip-add-support-for-veyron-speedy-ASUS-Chromebo.patch
+  patch -Np1 -i ../0002-rockchip-fix-incorrect-detection-of-ram-size.patch
+  patch -Np1 -i ../0003-Revert-doc-fix-incorrect-usage-of-DT-node-unit-addre.patch
+  patch -Np1 -i ../0004-rockchip-minnie-Add-a-.its-file-for-chromium.patch
+  patch -Np1 -i ../0005-rockchip-mickey-Add-a-.its-file-for-chromium.patch
+  patch -Np1 -i ../0006-rockchip-speedy-Add-a-.its-file-for-chromium.patch
+  patch -Np1 -i ../0007-veyron-set-CONFIG_SYS_TEXT_BASE-for-depthcharge.patch
+  patch -Np1 -i ../0008-dtoc-python2-compatibility.patch
```

Comment:
git apply works fine, and it's more strict.

Reply:
https://gist.github.com/hanetzer/c691e9618dd6b7f9e0420caac94b1261#file-stdin-L2450
Seems to indicate otherwise. Changing the 'git apply' to 'git apply -v' shows a lot of
'Skipped patch' messages.

Reply:
I'm not here to teach you how git works, and this isn't a debate.

Reply:
While I won't say 'I know how git works', I do use it fairly frequently. Changing the current
PKGBUILD to git apply -v results in the same skipped patches message and can't currently
build anyways.

https://gist.github.com/hanetzer/58ad46501e743407e0487be0a29aeec9

Reply:
https://asciinema.org/a/jT6sB7YhumZ1K2qLFXAYguYVd
0m35s: stock PKGBUILD, patches are skipped.
1m38s: my PKGBUILD using git apply, patches are skipped.
3m07s: my PKGBUILD using patch, patches are applied, build succeeds.

Reply:
@WarheadsSE ohhh, I just figured out what you mean here. Basically, because PKGBUILDs
is a git repo on my machine, when makepkg attempts to use git apply its thinking I'm trying for
the PKBUILDs repo and not the u-boot-${pkgver} directory, yes?